skycultures/tikuna/constellation_names.eng.fab

```diff
@@ -0,0 +1,4 @@
+001	"Baweta" _("Turtle Collective")
+002 "Wücütcha" _("Wücütcha’s")
+003 "Coyatchicüra" _("Cayman’s Jaw")
+004 "Tchatü-Ai" _("Anteater’s-Jaguar")

```

Comment:
should be without hyphen:
004 "Tchatü-Ai" _("Anteater’s Jaguar")

Reply:
ok I'm already making the changes